### PR TITLE
Test: Minor enhancements

### PR DESCRIPTION
**Pull request by facuspagnuolo**
This PR introduces multiple minor enhancements, it is recommended to review by commit

### CODE REVIEW

File: packages/test-helpers/src/decoding.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ async function decodeErrorReasonFromTx(tx, ctx) {
 }
 
 function decodeEvents(receipt, contractAbi, eventName) {
-  const rawLogs = receipt.rawLogs || []
+  const rawLogs = receipt.rawLogs || (receipt.receipt && receipt.receipt.rawLogs) || []
```
Comment (sohkai) [inline packages/test-helpers/src/decoding.js:43]: Oh interesting, how do you get nested receipts?

Should we propagate this change across all of the event helpers?

File: packages/test-helpers/src/decoding.js [hunk 2]
```diff
@@ -55,13 +55,14 @@ function decodeEvents(receipt, contractAbi, eventName) {
       log.data,
       log.topics.slice(1)
     )
+
     // Undo checksumed addresses
-    const eventArgs = Object.keys(decodedArgs).map((arg) =>
-      isAddress(arg) ? arg.toLowerCase() : arg
-    )
+    Object.entries(decodedArgs).forEach(([key, value]) => {
+      if (isAddress(value)) decodedArgs[key] = value.toLowerCase()
```
Comment (sohkai) [inline packages/test-helpers/src/decoding.js:61]: Surprising the linter didn't error on this, but let's use braces :)

File: packages/test-helpers/src/aragon-os/apps.js [hunk 3]
```diff
@@ -1,13 +1,9 @@
...
 
-function getInstalledApps(receipt, appIds, ctx) {
+function getInstalledApps(receipt, appIds) {
   appIds = Array.isArray(appIds) ? appIds : appIds ? [appIds] : null
 
-  const artifacts = getArtifacts(ctx)
-  const Kernel = artifacts.require('Kernel')
-
-  return decodeEvents(receipt, Kernel.abi, 'NewAppProxy')
+  return getEvents(receipt, 'NewAppProxy')
```
Comment (sohkai) [inline packages/test-helpers/src/aragon-os/apps.js:6]: I think we should still input the specific ABI here.

Maybe you did a call (e.g. to a template) where these weren't on the contract you directly called (and so the truffle contract doesn't know about the ABI).

File: packages/test-helpers/src/decoding.js (hunk 1 above)
Comment (facuspagnuolo) [reply packages/test-helpers/src/decoding.js:43]: Truffle returns a nested receipt when sending transactions like `receipt = { receipt, tx }`, while web3 returns a flat receipt when calling `eth.getTransactionReceipt` for example.

Comment (sohkai) [reply packages/test-helpers/src/decoding.js]: > Truffle returns a nested receipt when sending transactions like receipt = { receipt, tx }

Oh interesting, do you mean what the promise resolves to on contract invocations? This must be new truffle 5 behaviour?

**Pull request merged**